code/orders-infrastructure/src/main/java/com/academy/orders/infrastructure/order/entity/OrderEntity.java

```diff
@@ -1,24 +1,77 @@
 package com.academy.orders.infrastructure.order.entity;
 
-import java.util.UUID;
-
+import com.academy.orders.domain.order.entity.enumerated.OrderStatus;
+import com.academy.orders.infrastructure.account.entity.AccountEntity;
 import jakarta.persistence.Column;
+import jakarta.persistence.Embedded;
 import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.FetchType;
 import jakarta.persistence.GeneratedValue;
 import jakarta.persistence.GenerationType;
 import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
 import jakarta.persistence.Table;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.ToString;
+
+import static java.time.LocalDateTime.now;
 
-@Table
+@Table(name = "orders")
+@Entity
 @Data
 @Setter
-@Entity(name = "ORDER")
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode(exclude = {"account", "postAddress", "orderItems"})
+@ToString(exclude = {"account", "postAddress", "orderItems"})
 public class OrderEntity {
-
 	@Id
-	@Column(name = "ORDER_ID")
 	@GeneratedValue(strategy = GenerationType.UUID)
 	private UUID id;
+
+	@Column(name = "is_paid", nullable = false)
+	private Boolean isPaid;
+
+	@Column(name = "order_status", nullable = false)
+	@Enumerated(EnumType.STRING)
+	private OrderStatus orderStatus;
+
+	@Builder.Default
+	@Column(name = "created_at", nullable = false)
+	private LocalDateTime createdAt = now();
```

Comment:
You can use CreationTimestamp annotation. Also you can set updatable=false in Column annotation